app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here is the list of Posts for a given user</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag 'users_css' %>
+  </head>
+
+  <body>
+    <header>
+      <h1>This is the list of Posts for <%= @user.name %></h1>
+    </header>
+
+    <main>
+      <%= render 'users/user', user: @user %>
+      <h2>List of Posts:</h2>
+      <ul class="post-list flex-column">
+        <% @posts.each do |post| %>
```

Comment:
- [ ] Please makes sure that your app matches the given wireframe. Could you please remove the headings as they are not present in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB5V373VQQOELJYK76ML7IFAZGC5UMQ) ?✨🙌

Reply:
Done!